pkg/kubelet/kubelet.go

```diff
@@ -1274,6 +1274,9 @@ func (kl *Kubelet) cleanupOrphanedVolumes(pods []api.BoundPod, running []*docker
 
 // SyncPods synchronizes the configured list of pods (desired state) with the host current state.
 func (kl *Kubelet) SyncPods(pods []api.BoundPod, podSyncTypes map[types.UID]metrics.SyncPodType, start time.Time) error {
+	defer func() {
+		metrics.SyncPodsLatency.Observe(metrics.SinceInMicroseconds(start))
```

Comment:
Mentioned a bit offline, but I think we want to measure this raw. This is the interval at which we notice and enact changes. If this gets too long we will take long to respond. I think we want the non-normalized version.

It should also depend more on containers being deleted than number of pods since the syncing of pods is done async.